composer.json

```
@@ -22,7 +22,7 @@
     "ext-hash": "*",
     "ext-openssl": "*",
     "doctrine/common": "^2.5",
-    "firebase/php-jwt": "^6.0",
+    "firebase/php-jwt": "6.2",
```

```suggestion
    "firebase/php-jwt": "^6.2",
```

we should keep this pinned.